src/python_testing/matter_testing_support.py

```diff
@@ -847,6 +876,35 @@ async def send_single_cmd(
         result = await dev_ctrl.SendCommand(nodeid=node_id, endpoint=endpoint, payload=cmd, timedRequestTimeoutMs=timedRequestTimeoutMs)
         return result
 
+    async def send_test_event_triggers(self, enableKey: bytes = None, eventTrigger: int = 0x0099000000000000):
```

Comment:
Please do not default to a particular cluater

Reply:
Reordered params and removed.